scroll_menu.py

```diff
@@ -7,45 +11,88 @@
     CompletionsMenu,
     Float,
     FloatContainer,
+    FormattedTextControl,
     HSplit,
     Layout,
     ScrollablePane,
     VSplit,
+    Window,
 )
-from prompt_toolkit.widgets import Frame, Label, TextArea
+from prompt_toolkit.widgets import Button, Frame, Label
+
+
+class ScrollableMenu:
+    """Main class for scrollable menu"""
+
+    def __init__(self, dir: str):
+        # Display contents in the current working directory on the menu
+        cwd_content = os.listdir(dir)
+        frames = [
+            Frame(
+                Button(
+                    text=item,
+                    handler=functools.partial(self.display_content, item, "."),
```

Comment:
If `"."` is the directory path parameter in `self.display_content`, can we make that  a constant?